tests/test_maps.py

```diff
@@ -0,0 +1,167 @@
+import os
+import unittest
+from flask import jsonify
+from api import db
+from flask import Flask, request
+from api.models import Map
+from api.models import POI
+import requests
+from flask import jsonify
+import json
+
+# json objects to be tested 
+map_add_json_1 = {
+    "map_year" : 1669,
```

Comment:
in order to make sure your tests dont use maps that already exist, you can choose an arbitrary year, and keep making get requests until you find a map year that doesnt exist